### DISCUSSION TITLE
Game Version Updater for Modpacks

### DISCUSSION THREAD

**Discussion by TheDJr00mba** [modrinth/.github#305] [Feature requests]
The one thing I use modpacks for the most is to enhance my vanilla Realms experience. However I find it incredibly tedious any time the game gets an update that my Realms are forced to the latest vanilla release. 

Would it be possible to add a tool to the launcher that could update an older minecraft version's modpack to a newer minecraft version? 
For example, there are a lot of stable mods for 1.19.2 but vanilla got some cool stuff in 1.20.2, and lazy old me doesn't want to go looking for every single mod that I had in my 1.19.2 modpack in order to add it to my 1.20.2 modpack. 

Or maybe a better way to phrase it is, would it be possible to drop a modpack into this updater thingy and it tries to find all the mods in the current modpack but for the latest release of vanilla minecraft? It can even tell me that certain mods are unfound and it would just not add them.

@MagnarIUK Already has the same idea as me here https://github.com/orgs/modrinth/discussions/260

**Comment by MagnarIUK** [comment]
Heeey, glad I'm not the only one, who has this issue!

I, actualy, built myself a modpack updater tool, but it is broken since they changed data format to SQLite (and I didn't fixed it yet, because I can't find where they have put that database file

**Reply by TheDJr00mba** [reply]
I found the built-in updater option... Kinda disappointed that i didn't look any closer. Modpack > Options > Instance(Top setting box) > Edit mod loader/game versions > Edit Versions.

**Reply by TheDJr00mba** [reply]
![Screenshot 2024-12-05 142706](https://github.com/user-attachments/assets/565f14b9-fcac-401c-8a20-cd3c98966fd8)

**Reply by MagnarIUK** [reply]
It doesn't tho, it just changes version and then you need manually check and change every mod.
